pkgs/games/fabric-servers/derivation.nix

```diff
@@ -0,0 +1,60 @@
+{
+  stdenvNoCC,
+  fetchurl,
+  makeWrapper,
+  lib,
+
+  jre,
+  udev,
+
+  version,
+  url,
+  hash,
+  ...
```

Comment:
Package definitions that are callPackage'd should not take rest parameters due to their slightly ✨ magical ✨ nature. Leaving it out works just fine.

```suggestion
```